packages/rocketchat-ui-account/client/accountPreferences.js

```diff
@@ -152,19 +179,43 @@ Template.accountPreferences.onCreated(function() {
 		data.sendOnEnter = $('#sendOnEnter').find('select').val();
 		data.unreadRoomsMode = $('input[name=unreadRoomsMode]:checked').val();
 		data.roomsListExhibitionMode = $('select[name=roomsListExhibitionMode]').val();
-
 		data.autoImageLoad = $('input[name=autoImageLoad]:checked').val();
 		data.emailNotificationMode = $('select[name=emailNotificationMode]').val();
-		data.highlights = _.compact(_.map($('[name=highlights]').val().split(','), function(e) {
-			return s.trim(e);
-		}));
 		data.desktopNotificationDuration = $('input[name=desktopNotificationDuration]').val();
 		data.desktopNotifications = $('#desktopNotifications').find('select').val();
 		data.mobileNotifications = $('#mobileNotifications').find('select').val();
 		data.unreadAlert = $('#unreadAlert').find('input:checked').val();
 		data.notificationsSoundVolume = parseInt($('#notificationsSoundVolume').val());
-
 		data.roomCounterSidebar = $('#roomCounterSidebar').find('input:checked').val();
+		data.messageCounterSidebar = $('#messageCounterSidebar').find('input:checked').val();
```

Comment:
What is that item? I think it's old code, was replaced by `roomCounterSidebar`

Reply:
Must have been placed in the last merge conflict I needed to solve. Sorry about that.

Reply:
No problem, it was fixed